gensim/test/test_fasttext.py

```diff
@@ -1300,6 +1306,253 @@ def _run(self, fin):
         self.assertTrue(np.allclose(_ARRAY, array))
 
 
+def _create_and_save_test_model(fname, model_params):
```

Comment:
I think it's relevant to put `fb` in the function name to make it obvious which format you're saving in. `test` isn't particuarly relevant here.

```suggestion
def _create_and_save_fb_model(fname, model_params):
```

Reply:
Corrected.